### ISSUE TITLE
Parameters: { lamda, scale_pos_weight } might not be used.

### ISSUE THREAD

**Issue by ztz583** [dmlc/xgboost#5788]
environment: anaconda 4   xgboost 1.1.1
My code in like this:
param1 = {
          "objective":"multi:softmax"
          ,"num_class":7
          ,"max_depth":5
          ,"eta":0.05
          ,"gamma":0
          ,"lamda":2
          ,"alpha":1
          ,"n_jobs":-1
          ,"eval_metric":"merror"
            ,"colsample_bytree":0.5
            ,"colsample_bylevel":0.5
            ,"scale_pos_weight":2
            ,"base_score":0.9
          }
num_round= 100
n_fold=10
cvresult1 = xgb.cv(param1, dfull, num_round,n_fold)
Then gibe me the warning
WARNING: C:\Users\Administrator\workspace\xgboost-win64_release_1.1.0\src\learner.cc:480: 
Parameters: { lamda, scale_pos_weight } might not be used.

**Comment by hcho3** [comment]
* `lamda` is misspelled. It should be `lambda`.
* `scale_pos_weight` is only applicable for binary classification task. You are performing multi-class classification.

Take a look at the documentation: https://xgboost.readthedocs.io/en/latest/parameter.html

**Comment by ztz583** [comment]
> * `lamda`拼错了。应该是`lambda`.
> * `scale_pos_weight`仅适用于二进制分类任务。您正在执行多类分类。请移除`scale_pos_weight`.
> 
> 请看一下文档：https://xgboost.readthedocs.io/en/latest/parameter.html

How to deal with multi-class unbalanced samples？

**Comment by hcho3** [comment]
Assign more weights to minority classes.
